cradle-admin-tool-http/README.md

```diff
@@ -52,21 +52,22 @@ spec:
 
 ```
 Help:
- Description: this script provide ability to update comment for pages covered by time rage
+ Description: this script provide ability to update comment for pages covered by time rage where page.started is included and page.ended is excluded
  Required utils: jq curl paste grep head
  Arguments:
   --cradle-admin-tool-url (required) - cradle admin tool URL
   --book (required) - th2 book for searching and updating pages
-  --start-timestamp (required) - start timestamp for searching page to add --comment comment
-  --end-timestamp (required) - end timestamp for searching page to add --comment comment
-  --comment (required) - comment for adding to pages found from --start-timestamp to --end-timestamp
-  --mode (optional) - work mode. Default value is 'append'
+  --start-timestamp (optional) - start timestamp for searching page to add --comment comment. Default is min timestamp
+  --end-timestamp (optional) - end timestamp for searching page to add --comment comment. Default is max timestamp
```

Comment:
note: to be honest, this combination looks very dangerous... The start timestamp is not required, and the end timestamp is not required either. Users can easily update comments for all pages and don't even realize that...

suggestion: I would use some marker values, e.g. **newest** - to get the latest pages (it would be the max timestamp in our case), **oldest** - to get the oldest page value (it would be a min timestamp in our case), instead of defaulting to min/max values. It will be less dangerous for users

Reply:
Up to you. The main point, I think, is to require start and end timestamps for operations that modify something